shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/impl/OracleDMLVisitor.java

```diff
@@ -444,7 +445,7 @@ public ASTNode visitJoinSpecification(final JoinSpecificationContext ctx) {
         }
         if (null != ctx.USING()) {
             Collection<ColumnSegment> columnSegmentList = new LinkedList<>();
-            for (OracleStatementParser.ColumnNameContext cname :ctx.columnNames().columnName()) {
+            for (ColumnNameContext cname :ctx.columnNames().columnName()) {
```

Comment:
`ColumnNameContext cname :ctx.columnNames().columnName()` 
`cname` ---> `each`
A blank space is needed in front of `ctx.columnNames().columnName()`